cypress/integration/problem_detailed_page.js

```diff
@@ -19,4 +19,29 @@ describe("Problem Detailed Page", () => {
     cy.get(`a[href="/problems/1"]`).first().click();
     cy.contains("Pobierz treść");
   });
+
+  const path = require("path");
+
+  it("Initiate download pdf file containing problem details and verify it", () => {
+    cy.get(`a[href="/problems/1"]`).first().click();
+
+    cy.get('[class*="Button-sc-1v3p2bo-0 kuukQO"]')
+      .should("be.visible")
+      .click();
+    cy.wait(1000);
+    const downloadsFolder = Cypress.config("downloadsFolder");
+    cy.readFile(path.join(downloadsFolder, "Ciągi matematyczne.pdf")).should(
+      "exist"
+    );
+  });
+
+  it("Initiate download pdf file containing additional resources and verify it", () => {
+    cy.get(`a[href="/problems/1"]`).first().click();
+    cy.get('[class*="StyledButtonLink"]').should("be.visible").click();
```

Comment:
```suggestion
    cy.contains('additional resources').should("be.visible").click();
```

Reply:
I am not sure about that  - because "additional resources" at design has the name of the file. In that case it will not always be the same text.

Reply:
But it matches what we have in our mock data for the problem with id=1, unless we change mock data in handlers, it won't change. :)